server/models/userModel.js

```diff
@@ -2,16 +2,18 @@ const mongoose = require("mongoose");
 const jwt = require("jsonwebtoken");
 const config = require("config");
 
-
 const userSchema = new mongoose.Schema({
   name: { type: String, required: true },
   username: { type: String, required: true, unique: true },
   email: { type: String, required: true, unique: true },
   password: { type: String, required: true },
   location: { type: String },
   description: { type: String },
-  imageUrl: { type: String }
-
+  imageUrl: { type: String },
+  conversation: [{
```

Comment:
sorry a little pedantic, but 'conversation' should be 'conversations' because it's an array